tests/test_ratings.py

```diff
@@ -275,3 +275,137 @@ def test_flag_review_requires_login(selenium, base_url, variables):
             break
         else:
             count += 1
+
+
+@pytest.mark.nondestructive
+def test_flag_review_menu_options(selenium, base_url, variables):
+    extension = variables['all_scores_addon']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    addon.login('regular_user')
+    reviews = addon.ratings.click_all_reviews_link()
+    flag = reviews.review_items
+    count = 0
+    while count <= len(reviews.reviews_list):
+        if len(flag[count].review_body) > 0:
+            flag[count].flag_review.click()
+            # verifies that the following 3 report options are available in the flag menu
+            assert 'This is spam' in flag[count].flag_review_option[0].text
+            assert (
+                'This contains inappropriate language'
+                in flag[count].flag_review_option[1].text
+            )
+            assert (
+                'This is a bug report or support request'
+                in flag[count].flag_review_option[2].text
+            )
+            break
+        else:
+            count += 1
+
+
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_write_review_in_all_reviews_page(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    addon.login('regular_user')
+    # post a rating on the detail page
+    addon.ratings.rating_stars[4].click()
+    # navigate to the All reviews page to write your review
+    reviews = addon.ratings.click_all_reviews_link()
+    addon.ratings.write_a_review.click()
+    review_text = 'first review text'
```

Comment:
I am wondering if we can add these to the JSON variables file. Would help make this more modular

Reply:
@jrbenny35 I've replaced plain strings with variables in various places.